core/services/subscription.go

```diff
@@ -245,7 +243,12 @@ func timedUnsubscribe(unsubscriber Unsubscriber) {
 }
 
 func (sub ManagedSubscription) listenToLogs(q ethereum.FilterQuery) {
-	backfilledSet := sub.backfillLogs(q)
+	// If we spend too long backfilling without processing
+	// logs from our subscription, geth will consider the client dead
+	// and drop the subscription, so we set an upper bound on backlog processing time.
```

Comment:
What is geth's limit?

Reply:
```
// Slow subscribers will be dropped eventually. Client buffers up to 20000 notifications
// before considering the subscriber dead. The subscription Err channel will receive
// ErrSubscriptionQueueOverflow. Use a sufficiently large buffer on the channel or ensure
// that the channel usually has at least one reader to prevent this issue.
```

Reply:
Can we post this here, or maybe a link to it?